### PR TITLE
Fix/task drawer inputs

### PR DESCRIPTION
**Pull request by alexisdeh**
## Description

Update UI for task inputs in TaskDrawer

### CODE REVIEW

File: src/components/DrawerSection.tsx [hunk 1]
```diff
@@ -70,18 +71,26 @@ const DrawerSectionEntryWrapper = (props: StackProps) => (
...
     alignItems?: 'flex-start' | 'center' | 'baseline';
+    icon?: JSX.Element;
 }): JSX.Element => {
     return (
         <DrawerSectionEntryWrapper alignItems={alignItems || 'center'}>
-            <Text whiteSpace="nowrap" width="120px" flexShrink="0">
-                {capitalize(title)}
-            </Text>
+            {icon}
+            {titleStyle === 'code' && <Code fontSize="xs">{title}</Code>}
```
Comment (alexisdeh) [inline src/components/DrawerSection.tsx:88]: How about using this style also for the output of the parent_task (in TaskInputRepresentation, where we show 'parent_task_output_identifier') ?

**Review by alexisdeh — COMMENTED** [review]
Tested ok with latest backend and dump !

File: src/components/DrawerSection.tsx [hunk 2]
```diff
@@ -70,18 +71,26 @@ const DrawerSectionEntryWrapper = (props: StackProps) => (
 
 export const DrawerSectionEntry = ({
     title,
+    titleStyle,
     children,
     alignItems,
+    icon,
 }: {
     title: string;
+    titleStyle?: 'code';
```
[the remark this answers is not in the document]
Comment (Milouu) [reply src/components/DrawerSection.tsx:80]: Yes thought about it, didn't do it this way as 'capitalize' is just default and it's not gonna be used, but it's a bit clearer i agree.

File: src/components/DrawerSection.tsx [hunk 3]
```diff
@@ -116,9 +129,13 @@ export const DrawerSectionCollapsibleEntry = ({
                 cursor="pointer"
                 onClick={onToggle}
             >
-                <Text whiteSpace="nowrap" width="120px" flexShrink="0">
-                    {capitalize(title)}
-                </Text>
+                {icon}
```
Comment (HamdyD) [inline src/components/DrawerSection.tsx:132]: Not your code directly, but you may want to set `alignItems` to center now we have the icons (see screenshot with `flex-start`)

Capture d’écran 2022-09-22 à 16 58 31

Comment (Milouu) [reply src/components/DrawerSection.tsx:132]: nice catch

**Review by HamdyD — APPROVED** [review]

**Pull request merged**